apps/web/src/components/environment-access-guard.tsx

```
@@ -44,6 +47,15 @@ export function EnvironmentAccessGuard({
     syncUserThemePreferences,
   } = useThemeStore();
 
+  // Determine if user has access based on environment and feature flag
+  const envInfo = getEnvironmentInfo();
+  const isLocalhost = typeof window !== 'undefined' && 
```

ESLint

[error] Replace `·typeof·window·!==·'undefined'·&&·` with `⏎····typeof·window·!==·'undefined'·&&`

(prettier/prettier)